### PR TITLE
ARC-651 Increasing long query timeout to 5 seconds

### PR DESCRIPTION
**Pull request by mboudreau**
To prevent logging large JSON blobs in the logs, taking over DB storage.

### CODE REVIEW

File: github-for-jira.sd.yml [hunk 1]
```diff
@@ -138,7 +140,7 @@ workers:
 
 environmentOverrides:
 #  Uncomment lines 140-149 if you want remote debugging in ddev
-#  ddev:
+  ddev:
```
Comment (ccrolf) [inline github-for-jira.sd.yml:143]: Home come this is needed?

Comment (KAbakumovAtl) [reply github-for-jira.sd.yml:143]: I thought Amanda just fixed it in a separate PR. Maybe you need to merge main

Comment (KAbakumovAtl) [reply github-for-jira.sd.yml:143]: It was commented by accident

**Review by KAbakumovAtl — APPROVED** [review]

**Review by KAbakumovAtl — APPROVED** [review]

**Comment by atrigueiro** [comment]
Just adding the documentation from this change here, since it's not documented in Micros: [log_min_duration_statement](https://postgresqlco.nf/doc/en/param/log_min_duration_statement/)

**Pull request merged**